specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/Patches.json

```diff
@@ -0,0 +1,542 @@
+              "type": "string",
+              "description": "The details of the old layer for the target image."
+            },
+            "frameworkAndVersion": {
+              "type": "string",
+              "description": "The framework and its version in the old run image for the target image."
+            },
+            "osAndVersion": {
+              "type": "string",
+              "description": "The OS name and its version in the old run image for the target image."
+            }
+          },
+          "description": "The details of old run image for the target image.",
+          "x-nullable": true
+        },
+        "patchType": {
+          "type": "string",
+          "description": "The type for the patch.",
+          "enum": [
+            "FrameworkSecurity",
+            "OSSecurity",
+            "FrameworkAndOS",
+            "Other"
+          ],
+          "x-nullable": true,
+          "x-ms-enum": {
+            "name": "PatchType",
+            "modelAsString": true
+          }
+        }
+      },
+      "readOnly": true
+    },
+    "PatchProperties": {
+      "type": "object",
+      "properties": {
+        "targetEnvironmentId": {
+          "format": "arm-id",
+          "description": "The id of the target environment for the patch."
+        },
+        "targetContainerAppId": {
+          "format": "arm-id",
+          "description": "The id of the target container app for the patch."
+        },
+        "targetRevisionId": {
+          "format": "arm-id",
+          "description": "The id of the target revision for the patch."
+        },
+        "provisioningState": {
+          "$ref": "#/definitions/PatchProvisioningState",
+          "description": "Patch provisioning state.",
+          "readOnly": true
+        },
+        "patchApplyStatus": {
+          "$ref": "#/definitions/PatchApplyStatus",
+          "description": "The status of the patch once it has been provisioned",
+          "readOnly": true
+        },
+        "createdTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The UTC timestamp that describes when the patch object was created."
+        },
+        "updatedTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The UTC timestamp that describes when the patch object was last updated."
+        },
+        "patchDetails": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PatchDetails"
+          },
+          "description": "Detailed info about the specific patch.",
+          "x-ms-identifiers": []
+        }
+      },
+      "readOnly": true
+    },
+    "PatchCollection": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PatchResource"
+          },
+          "description": "The PatchResource items on this page"
```

Comment:
>  "description": "The PatchResource items on this page" [](http://example.com/codeflow?start=8&length=55)

consider a more human friendly description - a doc style review might be in order!